deploy/helm/sumologic/conf/metrics/metrics.conf

```diff
@@ -22,47 +22,120 @@
     @id sumologic.endpoint.metrics.apiserver
     endpoint "#{ENV['SUMO_ENDPOINT_METRICS_APISERVER']}"
     @include metrics.output.conf
+    <buffer>
+      {{- if eq .Values.sumologic.fluentdBuffer "file" }}
+        @type file
```

Comment:
indent looks off on these, e.g. https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/322/files#diff-77aa48b7684fe90d242ce9fdae9da300R58

I think they need to be something like
```
      {{- if eq .Values.sumologic.fluentdBuffer "file" }}
      @type file
      path /fluentd/buffer/metrics.kubelet
      {{- else }}
      @type memory
      {{- end }}
```

Reply:
good catch! Changed the indentation.